express-post-json/index.js

```diff
@@ -0,0 +1,33 @@
+const express = require('express')
+
+const app = express()
+
+let nextId = 1;
+
+const grades = {}
+
+
+app.get('/api/grades', function(req, res) {
+
+  const studentGrades = []
+  for (const student in grades){
+    studentGrades.push(grades[student])
+  }
+  res.json(grades)
+})
+
+
+app.use(express.json())
+
+app.post('/api/grades', function(req, res){
+  grades[nextId] = req.body;
+  grades[nextId].id = nextId;
+  const abc = nextId;
```

Comment:
Make sure to use descriptive variable names. Something like `currentId` would be good here